react-jsx/my-app/src/App.js

```diff
@@ -0,0 +1,28 @@
+import logo from './logo.svg';
+import './App.css';
+
+function MyButton() {
+  return <button>Click Me!</button>;
+}
+
```

Comment:
This shouldn't be a separate function. You should just have a `button` element down below in your `App` component.